src/graphql/__tests__/index.js

```diff
@@ -9,15 +9,10 @@ beforeEach(() => {
 });
 
 describe('getContext', () => {
-  it('generates null context for anonymous requests', async () => {
-    const context = await getContext({ ctx: { req: { headers: {} } } });
-
-    expect(context).toMatchInlineSnapshot(`
-      Object {
-        "userContext": null,
-        "userId": null,
-      }
-    `);
+  it('anonymous request fails with an error', async () => {
```

Comment:
`getContext` is used by GraphQL APIs that does not require authentication as well, such as `Query.insights`.

APIs that requires authentication are guarded by `@auth` directive, which detects empty context and throws `AuthenticationError`. `getContext`, on the other hand, should support anonymous requests.